### DISCUSSION TITLE
Laravel Pagination Merge

### DISCUSSION THREAD

**Discussion by aneeskhan47** [HassanZahirnia/laravel-package-ocean#131] [Package Suggestions]
Hey @HassanZahirnia , 

Thanks for the Laravel package ocean site. It's been beneficial for me. 

Would you be willing to add my package https://github.com/aneeskhan47/laravel-pagination-merge to the site as well? Let me know if that's possible. It can be added to the **Utilities & Helpers** category.

This package has **3.4K** installs and supports **Laravel 5, 6, 7, 8, 9, 10**

Thanks again!

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package!
